src/index.js

```diff
@@ -1,207 +1,206 @@
-import { Component } from 'react';
+import React from 'react';
```

Comment:
We don't have automated test yet on this project but if you want to do manual tests, you can run `npm run start` and play with the file located in `demo/src/index.js` by adding a `src` prop

Reply:
I have updated the tests in a new MR